### ISSUE TITLE
Ruby 2.7 deprecation warning with dsl/routing

### ISSUE THREAD

**Issue by dmke** [ruby-grape/grape#2140]
Basically the same as #1992/#2064, #2090:

```console
/home/app/shared/bundle/ruby/2.7.0/gems/grape-1.5.1/lib/grape/dsl/routing.rb:86: warning: Passing the keyword argument as the last hash parameter is deprecated
/home/app/shared/bundle/ruby/2.7.0/gems/grape-1.5.1/lib/grape/dsl/routing.rb:82: warning: The called method `mount' is defined here
```

**Comment by dmke** [comment]
I've just seen this is fixed in master (via #2091), can we have a new release?

**Comment by dblock** [comment]
Closed via #2091. Release when I or another maintainer has time.
